lib/huginn_acumen_product_agent/concerns/acumen_product_query_concern.rb

```diff
@@ -117,6 +118,7 @@ def parse_product_request(products)
             variant['offers'] = [{
                 '@type' => 'Offer',
                 'price' => field_value(p, 'Inv_Product.Price_1'),
+                'availability' => field_value(p, 'Inv_Product.BO_Reason')
```

Comment:
@Jhalter5Stones We should pass this availability to _both_ Offers so that things are consistent